### ISSUE TITLE
Male Infertility: Decreased sperm count

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9793]
**Preferred term label:**
Decreased sperm count

**Synonyms**

**Definition (free text, please give PubMed ID)**
≤39 million sperm (≤15 million/mL sperm concentration)

(World Health Organization. WHO laboratory manual for examination and processing of human semen. 6th Edition (2021))

**Parent term (use hpo.jax.org/app)**
HP:0012041

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 98048

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
Please relate new term requests to existing terms such as [Oligospermia](https://hpo.jax.org/app/browse/term/HP:0000798)  

Please comment on if we need a new parent, if the definition of the old term(s) needs revision, and how we should work on the hierarchy. For instance, to show a grandparent(A), parent(B), and children terms (C,D), you could write

term A
--term B
----term C
----term D

thanks!

**Comment by pnrobinson** [comment]
@MagoWyr 
The HPO tries not to "bundle" different concepts into the same term. Therefore, "Male infertility: Reduced sperm count" is not a good term because there are other causes of Male infertility.
If the suggestion is to create a term called Reduced sperm count (without "Male infertility", as is implied by the title), then I agree!

**Comment by MagoWyr** [comment]
"Decreased male fertility" and "Decreased sperm count" are two different terms. The latter is an offspring of the first. Please see the attached figure for illustration.
________________________________

**Comment by pnrobinson** [comment]
added/finalized per discussion
